sklearn/linear_model/least_angle.py

```diff
@@ -1437,7 +1438,7 @@ def fit(self, X, y, copy_X=True):
 
         self.alphas_ = alphas_
         with np.errstate(divide='ignore'):
-            self.criterion_ = n_samples * np.log(mean_squared_error) + K * df
+            self.criterion_ = n_samples * mean_squared_error/sigma + K * df
```

Comment:
ok agreed. can you see why travis is not happy and add a test that compares with R solution? thx